flake.nix

```diff
@@ -11,6 +11,7 @@
     nil_zkllvm_blueprint = {
       url = "https://github.com/NilFoundation/zkllvm-blueprint";
       type = "git";
+      rev = "15bec56f440aebe1a01d4c7a4229e5feb0e91041";
```

Comment:
I guess the blueprint workaround in this PR:

- https://github.com/NilFoundation/zkllvm-blueprint/pull/376

is coming soon, so maybe we will avoid checking out non-master revision here?

Reply:
I see, you outlined this in the description